pkg/tracing/scopedLogger.go

```diff
@@ -92,3 +99,17 @@ func (l scopedTraceLogger) Error(err error, attrs ...attribute.KeyValue) error {
 func (l scopedTraceLogger) Errorf(msg string, args ...interface{}) error {
 	return l.rootLogger.Errorf(l.span, msg, args...)
 }
+
+func callerFunc(frames int) string {
```

Comment:
While we don't need unit tests for the logs themselves, this is a great function to write a unit test for.

Reply:
Good idea! I will work on it in my next commit